Source/WebKit/webpushd/WebPushDaemonMain.mm

```diff
@@ -47,8 +47,15 @@
 using WebPushD::WebPushDaemon;
 
 static const ASCIILiteral entitlementName = "com.apple.private.webkit.webpush"_s;
+
+#if defined(RELOCATABLE_WEBPUSHD)
```

Comment:
I like #ifdef better, but this is fine.
Also, we might want to make this fit into the USE(RELOCATABLE_WEBPUSHD) convention we have elsewhere.